tika-parsers/src/main/resources/org/apache/tika/parser/recognition/tf/InceptionVideoRestDockerfile

```diff
@@ -15,37 +15,34 @@
 # limitations under the License.
 #
 
-# Take base as Ubuntu: 16.04, then you can simply apt-get install ffmpeg
 FROM ubuntu:16.04
 MAINTAINER Thamme Gowda <thammegowda@apache.org> and Kranthi Kiran GV <kkranthi@student.nitw.ac.in> and \
            Madhav Sharan <msharan@usc.edu> and Chris Mattmann <mattmann@apache.org>
 
+# Install missing part of ubuntu core + python + building dependencies
 RUN apt-get update && \
-      apt-get install -y wget && \
-      apt-get install -y unzip && \
-      apt-get install -y curl
-
-
-# Install ffmpeg
-RUN apt-get install -y ffmpeg
```

Comment:
maybe running apt-get install here forces the install of the dependencies you explicitly cite below (which is why it works still?)